scripts/referrals.ts

```diff
@@ -27,10 +17,11 @@ export function removeDuplicates(events: ReferralEvent[]): ReferralEvent[] {
   return Array.from(uniqueEventsMap.values())
 }
 
-// Fetch all referral events on all networks for the given protocol
+// Fetch referral events on networks for the given protocol with an optional list of referrer addresses
 export async function fetchReferralEvents(
   networkIds: NetworkId[],
   protocol: Protocol,
+  referrerAddresses?: string[],
```

Comment:
```suggestion
  referrerIds?: string[],
```
And same with the comment above